src/parser/lexer.h

```diff
@@ -252,12 +161,7 @@ struct Lexer {
   }
   TextPos position() const { return position(getPos()); }
 
-  size_t getPos() const {
-    if (curr) {
-      return getIndex() - curr->span.size();
-    }
-    return getIndex();
-  }
+  size_t getPos() const { return index; }
```

Comment:
Maybe worth renaming `index` to `pos` or vice versa?

Reply:
Yeah, we actually also have a `getIndex` method that does the same thing. Now that they are the same, it would be good to clean this up. I'll take a look at it as a follow-up.